### ISSUE TITLE
Adding more than one already added project to Codekit

### ISSUE THREAD

**Issue by rpkoller** [bdkjones/CodeKit2#167]
In order of preparing the screenshots of my other post i ran into the following. Only a cosmetic glitch but if you drag a few project folders onto Codekit which in part are already imported you have to click for each project the ok message. 

![bildschirmfoto 2014-01-18 um 10 31 07](https://f.cloud.github.com/assets/1665422/1946781/c08ffc0c-8023-11e3-84ce-6a5c3e48a7da.png)

Since it is just an informative alert panel i would suggest group the "already imported" alerts into a single alert panel. Won't happen often but just in case.

**Comment by bdkjones** [comment]
Doing that would be complex and time-consuming for very little gain. Don't add subfolders of projects.

**Comment by rpkoller** [comment]
Actually it wasn't a subfolder. I was testing with a normal folder. i had 14 folders on the desktop, each a small test case/project and added them . then i've added just a few a second time to see how things behave. like here: 

added folder
![bildschirmfoto 2014-01-19 um 05 25 49](https://f.cloud.github.com/assets/1665422/1948881/249ff6a4-80c2-11e3-805d-8e760985c08d.png)
then added the same folder again
![bildschirmfoto 2014-01-19 um 05 25 58](https://f.cloud.github.com/assets/1665422/1948882/2a0c07f4-80c2-11e3-80ac-351b305ef570.png)
that is what codekit is responding. 
![bildschirmfoto 2014-01-19 um 05 26 04](https://f.cloud.github.com/assets/1665422/1948883/2ebca1aa-80c2-11e3-93ab-2e73f37c420e.png)

so basically it isn't a subfolder like the alert panel text implies. it should just state it is already added to codekit. 

and since they aren't subfolders i still would think a combined alert panel for all project folders already imported would be neat. won't happen often but in the case the user will thank you only have to click once ok for an alert panel showing a list of projects already imported into codekit instead of for each duplicate project.
